CRM/Contribute/Form/ManagePremiums.php

```diff
@@ -55,11 +56,39 @@ public function setDefaultValues() {
       if (isset($tempDefaults['period_type'])) {
         $this->assign('showSubscriptions', TRUE);
       }
+
+      // Convert api3 field names to custom_xx format
+      foreach ($tempDefaults as $name => $value) {
+        if ($new = CRM_Core_BAO_CustomField::getShortNameFromLongName($name)) {
+          $tempDefaults[$new . '_' . $this->_id] = $value;
```

Comment:
what's up with appending `$this->id` to the custom field short name?

Reply:
This is for the default values, so it experts `custom_xxx_yy`  where `yy` is either the entity ID or `-1`.